File: src/main/java/com/zhongan/devpilot/completions/inline/OnCompletionPreviewUpdatedCallback.java
package com.zhongan.devpilot.completions.inline;

import com.zhongan.devpilot.completions.prediction.DevPilotCompletion;

public interface OnCompletionPreviewUpdatedCallback {
    void onCompletionPreviewUpdated(DevPilotCompletion completion);
}

File: src/main/java/com/zhongan/devpilot/completions/common/inline/OnCompletionPreviewUpdatedCallback.kt
package com.zhongan.devpilot.completions.common.inline

import com.zhongan.devpilot.completions.prediction.DevPilotCompletion

interface OnCompletionPreviewUpdatedCallback {
    fun onCompletionPreviewUpdated(completion: DevPilotCompletion)
}
